app/views/admin/_certificate.erb

```diff
@@ -0,0 +1,63 @@
+<% certificates.reverse.each do |certificate| %>
+  <table class="govuk-table">
+    <caption class="govuk-table__caption" id="certificates/<%= certificate.id %>">Certificate ID: <%= certificate.id %></caption>
+    <tbody class="govuk-table__body">
+      <tr class="govuk-table__row">
+        <th scope="row" class="govuk-table__header">Name</th>
+        <td class="govuk-table__cell"><%= certificate.x509.subject.to_s %></td>
+      </tr>
+      <tr class="govuk-table__row">
+        <th scope="row" class="govuk-table__header">Enabled</th>
```

Comment:
Should we be using localisation?

Reply:
I've decided not to use it in this instance, given it's a pure admin view.